generator/src/main/java/com/scottlogic/deg/generator/commandline/VisualiseCommandLine.java

```diff
@@ -51,6 +51,11 @@ public File getProfileFile() {
         return profileFile;
     }
 
+    @Override
+    public boolean disableSchema() {
```

Comment:
I's rename this to `disableSchemaValidation` or even `enableSchemaValidation` and invert/negate the field value.